Automattic/jetpack
# Instant Search: Make display order match tab order on mobile.

We do this by removing the CSS order attribute on the image. This means the image will be shown below the title instead of above it. For now, we should consider accessibility before styling.





Fixes #23367.

#### Changes proposed in this Pull Request:


Just removing the custom ordering from the CSS. If we want to put the image first in our mobile results, we should explore a way to do this that is accessible.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit a site with Jetpack Search enabled.
2. Open the search overlay with `?s=&result_format=expanded`.
3. Resize the window until the mobile view kicks in.
4. Confirm the tab order is: title, permalink, image.

![Composite](https://user-images.githubusercontent.com/40267301/183602658-76fd96c6-d709-4a9f-bebf-c65aa3a98104.png)
